src/BcPrebidVast.js

```diff
@@ -75,14 +79,24 @@ function dispatchPrebidDoneEvent() {
 
 // the function loads prebid.js and does header bidding if needed
```

Comment:
This function is sufficient complicated and also very important to do prebid in the header.  I think it needs a little more comments to explain all the steps.  In particular, what is the meaning of the fromHeader argument?  How is that set?  What kind of argument is it?  Also how does loadPrebidScript get invoked?